ketchup/watch_schedule_builder.py

```diff
@@ -0,0 +1,46 @@
+import datetime as dt
+import math
+
+
+def calculate_watch_schedule(episodes, end_date, start_date, per_day_limit=None):
+    """
+    There are some options on what we could return to the user:
+    1. basic episodes per day
+    2. limited number of episodes by end date (possible)
+    3. my limited number of episodes per day (not possible) which could return
+        a: basic eps per day
+        b: number of days to complete
+
+    :param episodes:
+    :param end_date:
+    :param start_date:
+    :param per_day_limit:
+    :return: response dict
+    """
+    response = {
+        "base_eps_per_day": 0,
+        "days": 0,
+        "actual_watch_days": 0,
+        "is_possible": True,
+        "realistic_end_date": None,
+    }
+
+    days_available = (end_date - start_date).days
+    eps_per_day = math.ceil(episodes / days_available)
+    actual_watch_days = episodes / eps_per_day
+    response["base_eps_per_day"] = eps_per_day
+    response["days"] = days_available
+    response["actual_watch_days"] = actual_watch_days
+
+    if per_day_limit:
+        # It's possible the per day limit resolves before the end date
+        min_days = math.ciel(episodes / per_day_limit)
```

Comment:
When attempting to run `ketchup.py` in my own venv, I get to this line and it says the following:

`
AttributeError: module 'math' has no attribute 'ciel'
`

Reply:
fixed